types/tedious-connection-pool/package.json

```
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/node": "*",
-        "@types/tedious": "*"
+        "@types/tedious": "^4"
```

```suggestion
        "@types/tedious": "*",
```
Please fix errors if any after reverting this change

---

Done